templates/web/fixamingata/report/new/form_report.html

```diff
@@ -0,0 +1,73 @@
+[% INCLUDE 'report/new/form_heading.html' %]
+
+[% IF field_errors.bodies %]
+    <p class='form-error'>[% field_errors.bodies %]</p>
+[% END %]
+
+[% PROCESS "report/new/category_wrapper.html" %]
+[% TRY %][% PROCESS 'report/new/after_category.html' %][% CATCH file %][% END %]
+
+    <h2 class="form-section-heading">Offentliga uppgifter</h2>
```

Comment:
This template exists because of this missed string (also 2.0.2), plus extra tips underneath the summary. So introducing a template hook (after merging this, does not have to hold this up) for those tips would mean this template wasn't needed either.